packages/@okta/vuepress-site/docs/guides/authenticators-okta-email/main/index.md

```diff
@@ -30,27 +33,143 @@ Learn how to integrate the Okta Email authenticator into your app with the Embed
+1. [Update Configurations](#update-configurations): Set up your org to enable the email authenticator and magic links
+1. [Integrate email challenge with magic links](#integrate-email-challenge-with-magic-links): Integrate email challenge using magic links with step-by-step instructions
+1. [Integrate different browser and device scenario with magic links](#integrate-different-browser-and-device-scenario-with-magic-links): Integrate the different browser and device scenarios with step-by-step instructions
+
+**OTP**
+
+1. [Integrate email enrollment with OTP](#integrate-email-enrollment-with-otp): Integrate email enrollment using OTP with step-by-step instructions
+1. [Enable only OTP for the email authenticator](#enable-only-otp-for-the-email-authenticator): Update the email templates to only support OTP
+
+**Advanced use cases**
+
+1. [Design considerations when customizing magic link for password recovery](#design-considerations-when-customizing-magic-link-for-password-recovery): Learn about recommended magic link implementations for specific customized password recovery solutions.
 
 ## Understand the magic link flow
 
-<StackSnippet snippet="understandmagiclinkflow" /> <!-- SHARED FOR BACKEND | NATIVE UNIQUE   -->
+Before integrating email magic links in your app, it's important to understand how your app's user journey starts and ends. An example user journey for a sign-in with email use case:
+
+1. Using your app, a user submits their username and password. Next, an email is sent to the user and your app displays an OTP input page.
+1. Using a new tab in the browser, the user opens their email and clicks on the magic link.
+1. In a new tab, the link redirects to your app and automatically signs the user in.
+
+The following diagram illustrates these steps.
+
+<div class="common-image-format">
+
+![Diagram showing magic link flow for same and different browsers](/img/authenticators/authenticators-email-magic-link-summary-user-flow-overview.png)
+
+</div>
+
+Note that the user is within the same browser when they start the sign-in and click on the magic link.
+
+### The reason for the OTP input page
+
+Your app should always display an OTP input page to allow the user to use the OTP as an alternative method to complete the verification. For security reasons, the user must use an OTP when they access their email on a different browser or device. See [Integrate different browser and device scenario](#integrate-different-browser-and-device-scenario-with-magic-links) to learn how to handle this scenario and [Integrate email enrollment with OTP](#integrate-email-enrollment-with-otp) to integrate OTP in your application.
+
+## Know which use cases support magic link and OTP
+
+Out of the box, the Embedded SDK solution supports magic links for various use cases. For those use cases where magic links is not fully supported, OTP is available. See the following support matrix for more details:
```

Comment:
Not changing. "Magic links" here is the feature so is singular.

Reply:
perhaps, since magic links is lowercase....might be hard to discern that it's a "feature"?

```suggestion
Out of the box, the Embedded SDK solution supports magic links for various use cases. For those use cases where the magic links feature isn't fully supported, OTP is available. See the following support matrix for more details:
```

Reply:
If it is the feature being referenced, then the full name is "Email Magic Link" (as per the PM spec https://docs.google.com/document/d/18xXSVzw4oCDpbeedzc1GFIa9Oo2oU2qc5fWuz8FfyKE/edit#heading=h.ldpelvtsnlbv) 

```suggestion
Out of the box, the Embedded SDK solution supports the Email Magic Link feature for various use cases. For those use cases where the feature isn't fully supported, OTP is available. See the following support matrix for more details:
```

Reply:
OK - Email Magic LInks it is